src/Serializer/Normalizer.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace App\Serializer;
+
+use ApiPlatform\Metadata\ApiProperty;
+use ApiPlatform\Serializer\ItemNormalizer;
+use App\Entity\ExcludeModification;
+use App\Entity\Member;
+use App\Entity\Payment;
+
+class Normalizer extends ItemNormalizer
+{
+/*    public function supportsNormalization(mixed $data, string $format = null, array $context = []): bool
+    {
+        return $data instanceof Payment
+            || $data instanceof Member
+            || $data instanceof ExcludeModification;
+    }*/
+
+    protected function normalizeRelation(
+        ApiProperty $propertyMetadata,
+        ?object $relatedObject,
+        string $resourceClass,
+        ?string $format,
+        array $context
+    ): \ArrayObject|array|string|null {
```

Comment:
### 🤓 Nitpick: Props order is fucked :)

The required props should always be at first plase, all optional - at the end of the list